### ISSUE TITLE
[a11y Design Review] Midpoint Prep Assistance: govcio-vfep (vebt): IBM Skillsbuild Training Program Intake Form (VA Form 22-10282) - (Figma Prototype)

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#90481]
# CAIA A11y Support

> [!NOTE]
> - **Review:** **FIGMA** - IBM Skillsbuild Training Program Intake Form (VA Form 22-10282)
> - **CAIA Review Date**: `8/13/2024` and `8/14/2024`
> - **Review Purpose**: Midpoint Review Prep
> - **Reviewed By**: CAIA A11ys 
> - **Midpoint Date:** `TBD`

CAIA A11ys can review the Figma prototype files for desktop and mobile (if available), in order to provide feedback that may help your team successfully prepare and pass your [Midpoint review ](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/midpoint-review)within the Collaboration Cycle.

## Next Steps

> [!TIP]
> **DEFINITION OF DONE**: Refer to the task list in this ticket, for actionable items.

## Description

The team went through Design Intent on July 15, 2024,  and is working on feedback from Brian (included in this ticket), with Midpoint being its next stop. 

### Timeframe
October/early fall project completion, per the intake ticket. 

### No Accessibility Specialist on Team
CAIA accessibility specialists are available to provide support to govcio-vfep (vebt) through this process with accessibility support, as this team does not have an accessibility specialist. 

## Details
_What details are necessary for understanding the specific work or request tracked by this issue?_

<details><summary>Toggle to view details from the intake ticket and design intent feedback</summary>

###  Intake Ticket

> - _Our initiative focuses on digitizing the current PDF form (22-10282) to enroll in IBM's free SkillsBuild training program._ 

> - _We're currently focusing on initial discovery and planning stages, alongside the early phases of digitization. Our primary efforts are centered around building and finalizing wireframes and developing a user flow that will guide our design intent for collaboration. While specific dates are still being refined, we're aiming for a prod push around October or early fall to ensure progress aligns with our timeline._

#### One-thing-per-page
Content Team (Katherine) has addressed [one-thing-per-page via comment and link.](https://github.com/department-of-veterans-affairs/va.gov-team/issues/87265#issuecomment-2246286920)

#### Heading Structure

> _**Should**: In the wireframes provided, some pages had an H3 heading (eg. "Employment and education") while others didn't (eg. the military affiliation page). A consistent heading structure on every page of the form makes it more predictable and easier to navigate for screen reader users, so if possible try to have an H3 on every page. There are times when the H2, the H3, and the question label are a little redundant, and that's okay. Try to phrase things a little differently so it's less repetitive ("military affiliation" and "Which military affiliation describes you best?" or something similar) as needed._

#### Hint Text or Additional Info. Components

> _**Should**: Wherever possible, provide hint text or additional info components providing users with context for why you're asking for some piece of information. It may not be clear to Veterans what the eligibility or selection criteria are for the program, and they may hit a question and think certain answers are disqualifying --- even if they're not. Any information that you can provide for why you're asking for employment information, education information, military affiliation, etc. can help reduce some anxiety some users may experience and help them understand the form and the process._

</details>

### Provided via CAIA Intake
- [CAIA Intake Ticket #87265](https://github.com/department-of-veterans-affairs/va.gov-team/issues/87265)
- [Product outline](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/veteran-education-benefits-tools/Forms/22-10282/Collab%20cycle/gcio-VEBT%20Form%2022-10282%20Product%20Outline.md)
- [Figma wireframes](https://www.figma.com/design/QTjUfSK6tYBSu7I1dYJaeO/VSAM-Forms_Wireframes?node-id=17-2&t=OAMYc8vaavn2kFsB-1) - `V2`
- [User flow](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1718211888145/62b605a599fa763fd3a91f46fb89b74f0748d1bb?sender=uc6c10065e2aa8f9e8a6a0717) Refer to the '22-10282' flow at the very top of the canvas

## CAIA Assignee(s)

<details><summary>Toggle to view assignee info</summary>
- @rileyorr 
- @coforma-jamie 
- @EvanAtCoforma 

</details>

## Team Info

<details><summary>Toggle to view team info</summary>

- **Team name:** govcio-vfep (switching to vebt soon)
- **OCTO product owner:** Darla van Nieukerk ([darla.vannieukerk@va.gov](mailto:darla.vannieukerk@va.gov))
- **Product name:** IBM Skillsbuild Training Program Intake Form (VA Form 22-10282)
- **Product manager**: Kara Ciprich ([kara.ciprich@va.gov](mailto:kara.ciprich@va.gov))
- **Slack channel:** [#govcio-vebt-team](https://join.slack.com/share/enQtNzMzMDg5NDYwNTQ5NC01MTUzZGU3MTNkNDBiMjUwOWJiMTllNTgyMzMwMTZjYzlkZDU2YjYxZjJiNTI2NWQ2NTg1M2Q0NTA5ZTc4MTll)
- **Dedicated content writer on your team (if you have one):** Garry Forbes @gforbes4 ([garry.forbes@va.gov](mailto:garry.forbes@va.gov))
- **Dedicated a11y specialist on your team (if you have one):** `None Listed`

</details>

## Definition of Done

```[tasklist]
### A11y Tasks
- [x] **CAIA Intake Opened** `6/28/2024` (with initial work being with content and IA - see comments in intake ticket for reference) #87265
- [x] **Review Design Intent Ticket** (feedback copied in this ticket's details section) #88550
- [x] Create a [Figma Prototype Review Google Doc](https://docs.google.com/document/d/1Q39d7lrTnOMQc4YDYD5Ieu9lqimdpGZyB18o77oE0b4/edit?usp=sharing) from the Figma Prototype Template
- [x] Riley's Review of the File
- [x] Jamie's Review of the File
- [x] Evan's Review of the File
- [x] **Review Figma Wireframes** post design intent, review [Figma wireframes](https://www.figma.com/design/QTjUfSK6tYBSu7I1dYJaeO/VSAM-Forms_Wireframes?node-id=17-2&t=OAMYc8vaavn2kFsB-1)
- [x] **Complete** the [Figma Prototype Review Google Doc](https://docs.google.com/document/d/1Q39d7lrTnOMQc4YDYD5Ieu9lqimdpGZyB18o77oE0b4/edit?usp=sharing) and share it with the govcio-vfep (vebt) team.
- [ ] **Additional Support** [Review the menu of services with the team](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/teams/CAIA/accessibility#services-we-offer).
- [ ] **Additional tickets** `TBD` - ex. research assistance, staging ticket assistance, etc.
```

```[tasklist]
## Ticket Updates (CAIA Internal)
- [x] _Connect to an `Epic` or `Intake` (what body of work is this a part of?)_
- [x] _Label with `Originator/Team` (product team or stakeholder requesting support)_
- [x] _Label date in the `Open Date` field_
- [x] _Label with `Estimate` (level of effort expected for this ticket)_
- [x] _Add `Assignee(s)` name(s) to ticket_
- [x] _Add `Assignee(s) name(s) to each task` they will complete via handle tag (if known)_
- [x] _Select a `Priority Level`_
- [x] _Update date in `Last Checked` field_
- [x] _Label with `Actual` (level of effort it took to complete this ticket)_
- [ ] _Update date in_ `Closed  Date`
```

**Comment by rileyorr** [comment]
The audit has been completed. The issues are documented below and can also be referred to in [the audit document](https://docs.google.com/document/d/1Q39d7lrTnOMQc4YDYD5Ieu9lqimdpGZyB18o77oE0b4/edit?usp=sharing). A comment will be added to the parent intake ticket directing the team to this feedback in addition to a post in their Slack channel. 

## Issue: Annotations are missing
**Screen:** All screens
**Version:** All
**Experience standard:** [User can complete necessary flows](https://design.va.gov/about/experience-standards/#user-can-complete-necessary-flows)
**Description:** [Annotations](https://design.va.gov/about/accessibility/accessibility-annotations) have not been included on the prototype
- Annotations help communicate accessibility decisions to other designers, developers, and the Platform Governance team.
- Annotations can help ensure that users with disabilities and people who use assistive technologies like screen readers have a smooth experience when using VA properties.
- Designers should add annotations before the design/developer hand-off. **The Collaboration Cycle strongly recommends you go to midpoint review with annotations** in your mockup files.

**Recommendation:** Consider adding annotations to the prototype prior to your team’s midpoint review
- If needed, you can reach out to CAIA to pair with an accessibility specialist on what annotations to add and how to add them. 

## Issue: Unclear language
**Screen:** “Need help?” area, all screens
**Version:** All
**Experience standard:** [User has enough information to complete a task](https://design.va.gov/about/experience-standards/#user-has-enough-information-to-complete-a-task)
**Description:** 
- The “Need help?” description is very short - it just says to Email a specific address. But it’s unclear how someone receiving that email can “help.” Are they providing tech support? Can they clarify eligibility? Or both? (See screenshot below)

![Screenshot 2024-08-14 at 12 28 32 PM](https://github.com/user-attachments/assets/086a230f-f4fb-417b-a446-1ee17e83c78b)

**Recommendation:** Add some language to clarify what kind of help [vettecpartner@va.gov](mailto:vettecpartner@va.gov) can provide. This may look something like: 
- “Email [vettecpartner@va.gov](mailto:vettecpartner@va.gov) if you have any questions about this program.” or
- “For help filling out this form, email vettechpartner@va.gov.”

## Issue: Error messages are missing and required elements are unclear
**Screen:** All pages with required inputs
**Version:** All
**Experience standard:** [User encounters consistent, clear design components and patterns](https://design.va.gov/about/experience-standards/#user-encounters-consistent-clear-design-components-and-patterns)
**Description:** Error messaging isn’t mocked up; it’s unclear what fields are optional vs required 
- The demographics portion of Section 1 states that the rest of the questions in the form are voluntary, which I perceive as meaning optional. However, Section 2 has a required input field “Enter the highest level of education you’ve completed”. This may be confusing to folks both with and without cognitive disabilities. See screenshots below. 

![Screenshot 2024-08-14 at 12 29 59 PM](https://github.com/user-attachments/assets/8c39a52f-709b-422d-92ef-db0ff2e52737)
![Screenshot 2024-08-14 at 12 30 14 PM](https://github.com/user-attachments/assets/e0fd1646-f7fd-4cf9-b601-29e97e9c7829)

- If the input field “Enter the highest level of education you’ve completed” is a required field when selecting “Something else” (there’s a bug in the prototype) then is the question itself required? Are other questions in Section 2 required?

**Recommendation:** 
- If the questions in Section 2 are required you may consider moving the demographic questions to a separate section at the end of the form. 
- If the questions in Section 2 are required, visually indicate they are required using the red (*Required) styling. 
- Consider mocking error states or documenting error messages so they can be reviewed as part of a future audit 

## Issue: Inconsistent component styling
**Screen:** Review submission screen
**Version:** All
**Experience standard:** [User encounters consistent, clear design components and patterns](https://design.va.gov/about/experience-standards/#user-encounters-consistent-clear-design-components-and-patterns)
**Description:** 
- The statement of truth component doesn’t align with the VA Design System statement of truth component. “Note: According to federal law,...” should not be displayed within the card.
![Screenshot 2024-08-14 at 12 34 18 PM](https://github.com/user-attachments/assets/1b0d6585-ffaa-4bc4-82a0-0eddd5fde845)


**Recommendation:** 
- Utilize [the Statement of truth USWDS component](https://design.va.gov/storybook/?path=/docs/uswds-va-statement-of-truth--docs) from the VA Design System.

## Issue: The review page is missing a heading
**Screen:** Review page
**Version:** All
**Experience standard:** [User encounters components and patterns that meet best practices](https://design.va.gov/about/experience-standards/#user-encounters-components-and-patterns-that-meet-best-practices)
**Description:** The review screen does not have a heading. All form pages should have a heading that provides context to the user about what the page is about.
![Screenshot 2024-08-14 at 12 34 38 PM](https://github.com/user-attachments/assets/b5e91ec4-f66e-4172-8f4e-2b2f34903fb5)


**Recommendation:** Add a heading describing the page’s purpose (ex. Review your application). There may be a standard heading used across all VA.gov forms that may be worth digging into.

**Comment by rileyorr** [comment]
I've added a comment in the parent ticket and reached out to the team in Slack 👍
